its/src/test/java/com/sonar/it/csharp/ProjectLevelDuplicationTest.java

```diff
@@ -36,18 +37,17 @@
 
 public class ProjectLevelDuplicationTest {
 
-  @Rule
-  public TemporaryFolder temp = TestUtils.createTempFolder();
+  @ClassRule
+  public static TemporaryFolder temp = TestUtils.createTempFolder();
 
-  @Before
-  public void init() {
-    TestUtils.reset(ORCHESTRATOR);
+  @BeforeClass
+  public static void init() throws IOException {
+    TestUtils.initLocal(ORCHESTRATOR);
+    Tests.analyzeProject(temp, "ProjectLevelDuplication");
```

Comment:
I'm not sure why this would happen in @BeforeClass and not the UT. There's just one UT anyway